package.json

```diff
@@ -82,12 +82,15 @@
     "dep-graph": "./node_modules/.bin/nx dep-graph",
     "workspace-schematic": "./node_modules/.bin/nx workspace-schematic",
     "help": "./node_modules/.bin/nx help",
-    "std-version": "standard-version --infile ./CHANGELOG.md --releaseCommitMessageFormat \"chore(release): version {{currentTag}} build ${TRAVIS_BUILD_NUMBER} [ci skip]\"",
     "nx": "nx",
     "postinstall": "ngcc --properties es2015 browser module main --first-only --create-ivy-entry-points",
     "docker-test": "docker run -u $(id -u) --rm -v \"$PWD\":/app trion/ng-cli-karma npm run test",
     "e2e-generate-report": "allure generate allure-results --clean -o allure-report",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "version.changelog": "run-s changelog git-add git-commit-changelog",
```

Comment:
@dimamarksman thank you bringing this. Since we have 5 repos which were done in the same way. Would you mind bringing this for the other 4:
- https://github.com/SAP/fundamental-styles
- https://github.com/SAP/fundamental-react
- https://github.com/SAP/fundamental-vue
- https://github.com/SAP/fundamental
this way everything is consistent

thank you

Reply:
@droshev to be honest mostly that's @Betrozov work.
I started making changes in the `contributing guide` here but found that there is one more PR regarding the `commit message lint`  from @Betrozov.
In order to avoid discrepancies, we agreed that he confluences his work here so all guide changes will be delivered as one PR.

Reply:
@Betrozov could you please bring these changes into mentioned projects?

Reply:
@droshev @dimamarksman 
Sure, will do it on Monday